### PR TITLE
replaced link images

### PR DESCRIPTION
**Pull request by vitagrazia81**

### CODE REVIEW

File: Leicester/Leicester210.xml [hunk 1]
```diff
@@ -329,6 +328,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
          <change who="VP" when="2022-09-26">Created catalogue entry stub</change>
       </revisionDesc>
    </teiHeader>
+   <facsimile>
+<graphic url="https://specialcollections.le.ac.uk/iiif/info/p15407coll6/20030/manifest.json"></graphic>
```
Comment (eu-genia) [inline Leicester/Leicester210.xml:332]: please here give the URI, not the manifest )

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: Leicester/Leicester210.xml (hunk 1 above)
Comment (eu-genia) [reply Leicester/Leicester210.xml:332]: (I know it is confusing. facs takes manifest, graphic url takes normal link to the landing page)

Comment (vitagrazia81) [reply Leicester/Leicester210.xml]: thanks @eu-genia. Url is there now.

**Review by eu-genia — APPROVED** [review]
thank you!

**Pull request merged**